build/bakefiles/files.bkl

```diff
@@ -360,7 +360,6 @@ IMPORTANT: please read docs/tech/tn0016.txt before modifying this file!
     src/qt/dcscreen.cpp
     src/qt/defs.cpp
     src/qt/dialog.cpp
-    src/unix/dialup.cpp
```

Comment:
Also need to modify `build/files` (from which this one is normally generated by running `build/upmake` -- which should also update CMake files).